plusdeck/client.py

```diff
@@ -42,15 +42,15 @@ class SubscriptionError(StateError):
 class Command(Enum):
     """A command for the Plus Deck 2C PC Cassette Deck."""
 
-    PlayA = b"\x01"
-    PlayB = b"\x02"
-    FastForward = b"\x03"
-    Rewind = b"\x04"
-    Pause = b"\x05"
-    Stop = b"\x06"
-    Eject = b"\x08"
-    Subscribe = b"\x0b"
-    Unsubscribe = b"\x0c"
+    PLAY_A = b"\x01"
+    PLAY_B = b"\x02"
+    FAST_A = b"\x03"
```

Comment:
I don't like that the commands are `FAST_X` while the corresponding states are `MOVING_FAST_X`. Maybe I can name them `FAST_MOVE_X` and `FAST_MOVING_X`?